packages/storybook/src/nimble/tree-view/tree-view.stories.ts

```diff
@@ -199,11 +201,20 @@ export const multipleTreeItems: StoryObj<TreeArgs> = {
             description:
                 'Event emitted when an item is selected or deselected.',
             table: { category: apiCategory.events }
+        },
+        leftMargin: {
+            name: 'Left margin',
+            description: 'Overrides the default left margin of the tree view.',
+            table: { category: apiCategory.styles },
+            control: { type: 'text' }
```

Comment:
I think having the token for configuration of this property is the right move but I don't know about broadcasting this so openly in the args table.

We could call it out in usage guidance in the mdx for the very specific use case. And we should have a test for the behavior. I'm actually tempted to have it as part of the matrix test to make sure it is interacting well with all the other modes that impact the spacing there (nested menu-items, icons, etc. )

Reply:
How about a compromise where we remove the docs from the args table and rely on the SWIF Chromatic tests to catch any regressions, instead of adding this to the matrix tests?

Reply:
I'd like nimble features to have tests in nimble. I wouldn't expect adding a new state to the matrix test to be difficult.
If don't want to bloat the current test matrix can split the dimension out into a separate test pretty easily.
If don't think it's worth testing against the full matrix can make a separate matrix test for the relevant permutations.

Reply:
Moving the PR back to draft until we decide on the testing direction